src/tests/ftest/util/environment_utils.py

```diff
@@ -408,7 +408,10 @@ def default_provider(self, logger, hosts):
         if result.passed:
             # Omni-Path adapter found; remove verbs as it will not work with OPA devices.
             logger.debug("  Excluding verbs provider for Omni-Path adapters")
-            supported.pop("verbs")
+            try:
+                supported.remove(VERBS_PROVIDER)
+            except ValueError:
+                pass
```

Comment:
This would be more dynamic
```suggestion
            supported = list(filter(lambda x: 'verbs' not in x, supported))
```